### DISCUSSION TITLE
Divergence of value(const value) AuxVariable in AuxKernel

### DISCUSSION THREAD

**Discussion by wangzhaohao** [idaholab/moose#21069] [Q&A General] [answered]
Hello, everyone
I want to ask is there have some ```Auxkernel``` about caculating the divergence of value? i find one about vector divergence. https://github.com/idaholab/moose/discussions/20554
my problem is konw the temperature field distribution, and want to use Auxkernel to get divergence of temp.

Thanks for your help in advance!

hao

**Comment by GiudGiud** [comment, marked as answer]
Hello

I made one in this PR https://github.com/idaholab/moose/pull/20956.
You can specify the same field for each component to do the divergence of a value
It's not merged yet, it ll take a few weeks probably. So you should just grab the file and add it to your MOOSE.

Please let me know how it goes. 

Guillaume
